docs/source/release_notes/v0.2.0.md

```diff
@@ -0,0 +1,33 @@
+# Version 0.2.0 (November 23, 2021)
+
+This is a new and enhanced version of MONAI Deploy App SDK, just in time for Thanksgiving and RSNA 2021!🎉
+
+Please visit [GETTING STARTED](/getting_started/index) guide and follow tutorials.
+
+You can learn more about SDK usage through [DEVELOPING WITH SDK](/developing_with_sdk/index).
+
+Please let us know how you like it and what could be improved by [submitting an issue](https://github.com/Project-MONAI/monai-deploy-app-sdk/issues/new/choose) or [asking questions](https://github.com/Project-MONAI/monai-deploy-app-sdk/discussions)😀
+
+## What's new in this version 0.2.0
+### Series Selection Operator
+This is to support the use case where whole DICOM studies are used as input to an AI inference application even though only specific series are applicable.
+
+The selection rules are defined in JSON, allowing multiple selections, each with a set of matching conditions. The rules processing engine is implemented in the `DICOMSeriesSelectorOperator`, which itself is regarded as a base class with default implementation. More advanced rules and processing engine can be implemented in derived classes.
+
+Multiple instances of the series selection operators, each having its own rules, can be chained in a MONAI Deploy application. In part this is made possible by the new App SDK Domain classes which encapsulate the selected series in a DICOM study, and are used as the output of each series selection operator.
+
+### DICOM Comprehensive Structured Report Writer
+This is introduced to support generating DICOM SR SOP instances for AI classification result, and as such, the DICOM SR writer is limited to support textual results.
+
+The DICOM SR writer is implemented in `DICOMTextSRWriterOperator`, it
+- loads AI result from in-memory object as well as from file path, with memory taking precedence
+- copies applicable DICOM tags from the original DICOM series used as input for the inference application, as well as generating tags anew when there is no DICOM series provided.
+- supports assigning DICOM tags via a dictionary with DICOM keywords and value, so that an application can customize the tags in the DICOM SR instance
```

Comment:
```suggestion
- supports assigning DICOM tags via a dictionary with DICOM keywords and values, so that an application can customize the tags in the DICOM SR instance
```

Reply:
value to their respective values, just to be specific.